pkg/transport/transport.go

```diff
@@ -105,12 +105,6 @@ func (transport *Transport) DialNode(ctx context.Context, node *pb.Node, opts ..
 		return nil, Error.Wrap(err)
 	}
 
-	ipAddr, err := getIP(conn.Target())
-	if err != nil {
-		return nil, err
-	}
-	node.LastIp = ipAddr
```

Comment:
I'm not sure if this part should be removed, I understand that the IP should be updated when the node is saved in the cache, however shouldn't we set the such filed when the node is dialed successful?

Reply:
@ifraixedes The IP is being filled further down the code in the alertSuccessful Method, as it is calling the cache PUT function. So it is called still when dialing the node.

Reply:
always? or it depends of the observers?

Reply:
We were depending on the observers before already.